### DISCUSSION TITLE
Can I use 'VectorScene' with manim-slides?

### DISCUSSION THREAD

**Discussion by marisbaier** [jeertmans/manim-slides#209] [Q&A] [answered]
I'm trying to animate a vector space
```python
from manim import *
from manim_slides import *

class Animation(VectorScene):
    def construct(self):
        plane = self.add_plane(animate=True).add_coordinates()
        vector = self.add_vector([-3,-2], color = YELLOW)
```
Can I even do that with manim-slides because it has only the 'Slide'-class?

**Comment by jeertmans** [comment, marked as answer]
Hello @marisbaier, thank you for your question!

So the basic answer is: create a new class that subclass `Slide` and `VectorScene`.

There is a specific section in the docs explaining how to do that: [Subclass Custom Scenes](https://eertmans.be/manim-slides/reference/examples.html#subclass-custom-scenes).

Here is a screenshot of its actual content:

![image](https://github.com/jeertmans/manim-slides/assets/27275099/7db01b2b-1924-412c-8126-dc5ca832e2d0)

I hope it helps!
